README.md

```diff
@@ -1,35 +1,90 @@
 # cvs-svc-technical-records
+Please install and configure [nvm](https://github.com/nvm-sh/nvm) with this node version so it can be used during installation.
+
+
+## Architecture
+
+To capture and persist technical records for a Vehicle submitted by the client (VTM) which is then made available to VTA/VTM for searching a vehicle. If the submitted technical record is applicable for a ministry plate certificate, it triggers the generation of the certificate via dynamo streams of the technical results table.
+
+### End to end design
+[All in one view](https://wiki.dvsacloud.uk/pages/viewpage.action?pageId=79254695)
+
+![All in one](/docs/allinone.png)
```

Comment:
I think just having the link to confluence is sufficient. Including the image for the, all in one view isn't required, seems quite a lot in one picture, which isn't required for this specific microservice. Also, developer wont be updating the all in one view(architect would), so better single version of truth is maintained in confluence. All other diagrams on this readme are fine as they are specific to this microservice and being low level developers should be maintaining that.

Reply:
good suggestion, I agree, doing the changes.
I found that a picture could make things look a bit better in the README and help a bit more.